src/adapters/supabase/helpers/wallet.ts

```diff
@@ -21,8 +21,7 @@ export class Wallet extends Super {
   public async upsertWalletAddress(context: Context, address: string) {
     const payload = context.payload;
 
-    const userData = await this._getUserData(payload);
-    const registeredWalletData = await this._getRegisteredWalletData(userData);
+    const registeredWalletData = await this._getRegisteredWalletData(address);
```

Comment:
I misunderstood then, I thought we should support multiple accounts tied to the same wallet, otherwise I don't see the point of the `unset` option. I will change it back.

Reply:
Fixed in https://github.com/ubiquity-os-marketplace/command-wallet/pull/29/commits/dd47e9cbba77d4be0dbded89d34d74c59a61beae